docs/antrea-network-policy.md

```diff
@@ -272,6 +274,22 @@ single port, to the 10.0.10.0/24 subnet specified by the `ipBlock` field.
 **Note**: The order in which the egress rules are set matter, i.e. rules will
 be enforced in the order in which they are written.
 
+**enableLogging**: A ClusterNetworkPolicy ingress or egress rule can be
+audited by enabling its logging field. When `enableLogging` field is set to
+true, the first packet of any connection that matches this rule will be logged
+to a separate file (`/var/log/antrea/networkpolicy/np.log`). These log files
```

Comment:
We should mention log entries will only be added to this file on the that node/agent for appliedTo pods that resides on the same node.

Reply:
updated